src/otx/algorithms/anomaly/tasks/train.py

```diff
@@ -67,7 +67,7 @@ def train(
         if seed:
             logger.info(f"Setting seed to {seed}")
             seed_everything(seed, workers=True)
-        config.trainer.deterministic = "warn" if deterministic else deterministic
+        config.trainer.deterministic = deterministic
```

Comment:
I modified this at https://github.com/openvinotoolkit/training_extensions/pull/2361 to avoid errors about operations that deterministic doesn't support.
I'd like to know why you revert this part.